cli/flox/doc/flox-update.md

```diff
@@ -0,0 +1,56 @@
+---
+title: FLOX-UPDATE
+section: 1
+header: "flox User Manuals"
+...
+
+
+# NAME
+
+flox-update - update the global base catalog or an environment's base catalog
+
+# SYNOPSIS
+
+```
+flox [ <general-options> ] update
+     [--global | (-d=<path> | -r=<owner/name>)]
+```
+
+# DESCRIPTION
+
+Update the global base catalog if `--global` is specified,
+or update an environment's base catalog.
+
+The base catalog is a collection of packages used by various flox subcommands.
+
+The global base catalog provides packages for
+[`flox-search(1)`](./flox-search.md) and [`flox-show(1)`](./flox-show.md) when
+not using an environment,
+and it is used to initialize an environment's base catalog.
```

Comment:
```suggestion
The global base catalog is used by [`flox-init(1)`](./flox-init.md)
to initialize an environment's base catalog
and provides packages for
[`flox-search(1)`](./flox-search.md), and [`flox-show(1)`](./flox-show.md)
when not using an environment.
```

Reply:
We may want to change the behavior, but the global catalog isn't used until install time. E.g. if you `flox init && flox update --global && flox install`, you'll get the updated catalog